WorkflowUI/Tests/ViewEnvironmentTests.swift

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2021 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if canImport(UIKit)
+
+    import XCTest
+
+    @testable import WorkflowUI
+
+    class ViewEnvironmentTests: XCTestCase {
```

Comment:
Nit: It would be nice to test a couple layers of nesting to ensure that works as you'd expect